models/issue_tracker.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/VojtechVitek/go-trello"
+	"github.com/google/go-github/github"
+	"log"
+	"strings"
+)
+
+type Configuration struct {
+	ApiKey   string
+	Token    string
+	UserName string
+}
+
+type TrelloIssueProvider struct {
+	Configuration
+	BoardId  string
+	ListName string
+}
+
+type GithubIssueProvider struct {
+	Query string
+}
+
+type Issue struct {
+	title       string
+	description string
+}
+
+func PrintIssue(issue Issue) {
+	fmt.Println("title: ", string(issue.title))
+	fmt.Println("issue: ", string(issue.description))
+	fmt.Println("")
+}
+
+func (t TrelloIssueProvider) FetchData(c chan Issue) {
+	trello, err := trello.NewAuthClient(t.Configuration.ApiKey, &t.Configuration.Token)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// @trello Boards
+	board, err := trello.Board(t.BoardId)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// @trello Board Lists
+	lists, err := board.Lists()
+	if err != nil {
+		log.Fatal(err)
+	}
+	for _, list := range lists {
+		if strings.Compare(list.Name, t.ListName) == 0 {
+			// @trello Board List Cards
+			cards, _ := list.Cards()
+			for _, card := range cards {
+				cardName := card.Name
+				description := card.Desc
+				issueInstance := Issue{cardName, description}
+				c <- issueInstance
+			}
+			close(c)
+		}
+	}
+}
+
+func (g GithubIssueProvider) FetchData(c chan Issue) {
+	client := github.NewClient(nil)
+	opts := &github.SearchOptions{
+		ListOptions: github.ListOptions{
+			PerPage: 100,
+		},
+	}
+
+	result, _, err := client.Search.Issues(g.Query, opts)
+	issues := result.Issues
+	for l, _ := range issues {
+		url := issues[l].URL
+		title := issues[l].Title
+		issueInstance := Issue{*title, *url}
+		c <- issueInstance
+	}
+	close(c)
```

Comment:
I think we should change it around and have FetchData return a chan Issue, as oppose to receive one as an argument. That clears up the confusion of why this method is closing 'someone' else's channel.